uiplib/utils/utils.py

```diff
@@ -42,3 +44,13 @@ def check_sites(settings):
                 else:
                     sites_present[key] = True
     return sites_present
+
+
+def flush_wallpapers(settings):
+    """Delete all downloaded wallpapers."""
+    print("Deleting all downloaded wallpapers...")
+    try:
+        shutil.rmtree(settings['pics-folder'])
```

Comment:
i know this is how it was done earlier, but can we have something else? atleast a warning?